prisma/migrations/20230811101958_rename_to_pat/migration.sql

```diff
@@ -0,0 +1,26 @@
+/*
+  Warnings:
+
+  - You are about to drop the `PTA` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- DropForeignKey
+ALTER TABLE "PTA" DROP CONSTRAINT "PTA_userId_fkey";
+
+-- DropTable
+DROP TABLE "PTA";
+
+-- CreateTable
+CREATE TABLE "PAT" (
```

Comment:
ok, now I see - but again I would name it `PersonalAccessTokens` - when you dive in into the code for the first time you might not know if PAT is PersonalAccessToken or Port Address Translation or Portland Association of Teachers